frontend/src/features/env/SwitchEnvFeedbackModal.tsx

```diff
@@ -147,82 +146,116 @@ export const SwitchEnvFeedbackModal = ({
-                          {...register(FEEDBACK_OTHERS_INPUT_NAME, {
-                            validate: (value) => {
-                              return (
-                                getValues('switchReason') !==
-                                  othersInputValue ||
-                                !!value ||
-                                'Please specify a value for the "Others" option'
-                              )
-                            },
-                          })}
-                        />
-                      </FormControl>
-                    </Radio.OthersWrapper>
-                  </Radio.RadioGroup>
+                        <FormControl>
+                          <OthersInput
+                            aria-label='"Other" response'
+                            {...register(FEEDBACK_OTHERS_INPUT_NAME, {
+                              validate: (value) => {
+                                return (
+                                  getValues('switchReason') !==
+                                    othersInputValue ||
+                                  !!value ||
+                                  'Please specify a value for the "Others" option'
+                                )
+                              },
+                            })}
+                          />
+                        </FormControl>
+                      </Radio.OthersWrapper>
+                    </Radio.RadioGroup>
+                    <FormErrorMessage>
+                      {errors['switchReason']?.message ??
+                        errors[FEEDBACK_OTHERS_INPUT_NAME]?.message}
+                    </FormErrorMessage>
+                  </FormControl>
+                ) : null}
+                <FormControl
+                  isRequired={!isAdminView}
+                  isInvalid={!isAdminView && !isEmpty(errors)}
+                >
+                  <FormLabel
+                    description={
+                      isAdminView
+                        ? ''
+                        : 'Any fields you’ve filled in your form so far will be cleared'
+                    }
+                  >
+                    {isAdminView
+                      ? 'Describe your problem in detail to help us improve FormSG'
+                      : 'Please tell us about the issue(s) you’re facing. It’ll help us improve FormSG.'}
+                  </FormLabel>
+                  <Textarea
+                    {...register('feedback', {
+                      required: {
+                        value: !isAdminView,
+                        message: 'This field is required',
+                      },
+                    })}
+                  />
                   <FormErrorMessage>
-                    {errors['switchReason']?.message ??
-                      errors[FEEDBACK_OTHERS_INPUT_NAME]?.message}
+                    {console.log(errors['feedback']?.message)}
+                    {errors['feedback']?.message}
```

Comment:
Question: is the console log to help us debug for users (but wouldn't message already be displayed on the form?)

Reply:
haha whoops no, forgot to remove it. removed it [here]([404faa9](https://github.com/opengovsg/FormSG/pull/5339/commits/404faa9ca5e8e75591977036f97c9eaa97a52822)) :)